javascript/ql/src/semmle/javascript/dataflow/Nodes.qll

```diff
@@ -89,9 +90,24 @@ class InvokeNode extends DataFlow::SourceNode {
 
   Function getEnclosingFunction() { result = getBasicBlock().getContainer() }
 
-  /** Gets a function passed as the `i`th argument of this invocation. */
+  /**
+   * Gets a function passed as the `i`th argument of this invocation.
+   *
+   * This predicate only performs local data flow tracking.
+   * Use `getACallbackSource` to handle interprocedural flow of callback functions.
+   */
   FunctionNode getCallback(int i) { result.flowsTo(getArgument(i)) }
 
+  /**
+   * Gets a function passed as the `i`th argument of this invocation,
```

Comment:
There is no parameter `i`, and the meaning of `param` isn't documented. I also think this predicate is complex enough to merit a brief example in the doc comment.